### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by feki-rihab**
Standardization level from 3 to 2
Datasets publisher updates & time stamps

### CODE REVIEW

File: Datasets/Cyprus_Pilot/consumer_reports_metadata.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   ],
   "accessRights": "",
   "creator": [
-    "CY Pilot"
+    "CERTH"
```
Comment (flopezag) [inline Datasets/Cyprus_Pilot/consumer_reports_metadata.json:37]: PHOEBE

**Review by flopezag — CHANGES_REQUESTED** [review]

File: Datasets/Cyprus_Pilot/consumer_reports_metadata.json (hunk 1 above)
Comment (feki-rihab) [reply Datasets/Cyprus_Pilot/consumer_reports_metadata.json:37]: But this is part of the Cypus Pilot, so I should keep it CERTH and just change the person's name, or?

Comment (flopezag) [reply Datasets/Cyprus_Pilot/consumer_reports_metadata.json:37]: Better to put PHOEBE for the moment and then we can change it, just to close it

Comment (feki-rihab) [reply Datasets/Cyprus_Pilot/consumer_reports_metadata.json:37]: Alright!

**Review by flopezag — APPROVED** [review]

**Pull request merged**